tests/test_spherical.py

```diff
@@ -0,0 +1,65 @@
+"""
+Tests ...
+"""
+from pytest import raises
+import numpy as np
+import spharpy.spherical as sh
+
+
+def test_renormalize():
+    sh_data = np.ones((4, 2))
+    current_norm = 'n3d'
+
+    # test invalid type
+    with raises(ValueError, match="Invalid normalization. Has to be 'sn3d', "
+                                  "'n3d', or 'maxN', but is wrong_norm"):
+        sh.renormalize(sh_data, 'acn', current_norm, 'wrong_norm', axis=0)
+
+    # test maxN
+    sh_data_renorm = sh.renormalize(sh_data.copy(), 'acn', current_norm,
```

Comment:
It's maybe a little dangerous to require the user to copy the data if it should now be overwritten. In line with other pyfar functions, I would suggest to take the copy inside the functions
```suggestion
    sh_data_renorm = sh.renormalize(sh_data, 'acn', current_norm,
```